src/utils/components/DiskModal/components/DiskTypeSelect/DiskTypeSelect.tsx

```diff
@@ -46,8 +52,10 @@ const DiskTypeSelect: FC<DiskTypeSelectProps> = ({ isVMRunning }) => {
             setValue('disk.disk', null);
 
             // cdrom does not support virtio
+            // lun only supports scsi
```

Comment:
nitpick: You can make this one comment:
`// cd-rom doesn't support virtio and LUN only supports SCSI`